.github/workflows/ci.yaml

```diff
@@ -1,6 +1,11 @@
 name: CI
 on:
   push:
+
+permissions:
```

Comment:
unused in this workflow because there's no reference to ${{secrets.BLAHBLAH}} here